clang/test/SemaCXX/alias-template.cpp

```diff
@@ -65,7 +65,8 @@ namespace InFunctions {
   template<typename...T> struct S3 { // expected-note {{template parameter is declared here}}
     template<typename Z> using T = int; // expected-error {{declaration of 'T' shadows template parameter}}
   };
-  template<typename Z> using Z = Z;
+  template<typename Z> // expected-note {{template parameter is declared here}}
+  using Z = Z; // expected-error {{declaration of 'Z' shadows template parameter}}
```

Comment:
I am not sure that `InFunctions` is the correct namespace for this test but it was here before and I adapted this use case to the new behavior.

Reply:
That namespace name is weird -- there are no functions in it at all. There doesn't even seem to be any common theme for these tests. It'd be nice to split this namespace up into distinct namespaces for the different things being tested -- maybe `UnexpandedPack` for `S0`, `InvalidType` for `T1` and `T2`, and `ShadowTemplateParam` for `S3` and `Z`.